### PR TITLE
Adds detection for AOC, ALDI NORD, ELECTRONIA, Facebook, F&U, Fuego, GoGEN, Horizon, KUBO, Laurus, Lifemaxx, Manhattan, MStar, OK, PROFiLO, REGAL, SCBC, SEG, SWTV, Technicolor, Technika, TOKYO, WELLINGTON, Western Digital, VOX television brands, improves detection for Akai, ARRIS, Blaupunkt, Essentielb, Haier, Hi-Level, Hisense, Hitachi, Hyundai, Kalley, LG, MediaTek, Miray, MTC, Nvidia, Panasonic, Philips, Qilive, Samsung, Smart, Sony, TCL, TD Systems, Thomson, Vestel television brands and improves detection for Android, Fire OS, webOS

### PR DESCRIPTION
**Pull request by liviuconcioiu**

### CODE REVIEW

File: Tests/fixtures/tv.yml [hunk 1]
```diff
@@ -6966,3 +6966,2506 @@
...
+  client:
+    type: browser
+    name: Opera Devices
+    version: 4.6.1.40
+    engine: Blink
+    engine_version: ""
+  device:
+    type: tv
+    brand: Electronia
+    model: MB110
```
Comment (sanchezzzhak) [inline Tests/fixtures/tv.yml:7812]: ELECTRONIA brand may exist

often `MB[0-9]{3,4}` are not models, it is a common MainBoard/firmware. Moreover, these symbols are found for different brands 

http://www.zeppy.io/discover/gb/mb110

Comment (liviuconcioiu) [reply Tests/fixtures/tv.yml]: Should I also correct some regexes that start with LF as model name? As far as I can see is a firmware version, and not a model. Right now are seen as models.
`Opera/9.80 (Linux mips; U; NETRANGEMMH;HbbTV/1.1.1;CE-HTML/1.0;TCL LF1V048; en) Presto/2.10.287 Version/12.00
Opera/9.80 (Linux armv7l; U; NETRANGEMMH;HbbTV/1.1.1;CE-HTML/1.0;Vendor/THOMSON;SW-Version/V8-MT51F01-LF1V466;Cnt/FRA;Lan/eng) Presto/2.12.362 Version/12.11`

Comment (sanchezzzhak) [reply Tests/fixtures/tv.yml]: > Should I also correct some regexes that start with LF as model name? As far as I can see is a firmware version, and not a model. Right now are seen as models.
> `Opera/9.80 (Linux mips; U; NETRANGEMMH;HbbTV/1.1.1;CE-HTML/1.0;TCL LF1V048; en) Presto/2.10.287 Version/12.00`
> `Opera/9.80 (Linux armv7l; U; NETRANGEMMH;HbbTV/1.1.1;CE-HTML/1.0;Vendor/THOMSON;SW-Version/V8-MT51F01-LF1V466;Cnt/FRA;Lan/eng) Presto/2.12.362 Version/12.11`

yes replace

Comment (sanchezzzhak) [reply Tests/fixtures/tv.yml]: https://www.hbbtv.org/resource-library/specifications/
I have noticed that if the habtv does not meet the specification, then it is the motherboard or firmware.
Often such useragents contain the DOM2-3 prefixes

example 

`Mozilla/5.0 (Linux armv7l) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/53.0.2785.143 Safari/537.36 OPR/40.0.2207.0 OMI/4.9.0.237.DOM3.223 Model/Vestel-MB211 VSTVB MB200 HbbTV/1.2.1 (; GOGEN; MB211; 0.59.2.2; _TV_NT72563_2017;) SmartTvA/3.0.0
`  prefix  `DOM3`

File: regexes/device/televisions.yml [hunk 2]
```diff
@@ -74,23 +104,79 @@ DMM:
...
+  model: ''
+
 # Finlux
 Finlux:
   regex: 'FINLUX[;,]'
   device: 'tv'
   model: ''
 
+# FU
+FU:
```
Comment (sanchezzzhak) [inline regexes/device/televisions.yml:126]: name brand `F&U`
official site
https://www.fandu.gr/categories.asp?catid=246&title=tv-sets

**Review by sanchezzzhak — CHANGES_REQUESTED** [review]

File: regexes/oss.yml [hunk 3]
```diff
@@ -150,6 +154,14 @@
   name: 'Android'
   version: ''
 
+- regex: '([24])K_Android_TV_V01|UMC_2KAndroidTV_2019'
+  name: 'Android'
+  version: ''
```
Comment (sanchezzzhak) [inline regexes/oss.yml:159]: combine to up rule
```yml
- regex: 'BeyondPod|AntennaPod|Podkicker|DoggCatcher|Player FM|okhttp|Podcatcher Deluxe|.+K_?Android_?TV_'
  name: 'Android'
  version: ''
```

**Review by sanchezzzhak — CHANGES_REQUESTED** [review]

File: Parser/Device/AbstractDeviceParser.php [hunk 4]
```diff
@@ -741,7 +759,9 @@ abstract class AbstractDeviceParser extends AbstractParser
         'SQ' => 'Santin',
         'SY' => 'Sanyo',
         'S9' => 'Savio',
+        'Y4' => 'SCBC',
```
Comment (sanchezzzhak) [inline Parser/Device/AbstractDeviceParser.php:762]: I checked everything, but this brand is not located at all

**Review by sanchezzzhak — DISMISSED** [review]

File: Parser/Device/AbstractDeviceParser.php (hunk 4 above)
Comment (sanchezzzhak) [reply Parser/Device/AbstractDeviceParser.php:762]: @sgiehl can we do a merge with the master without confirming the brand `SCBC` ?
@liviuconcioiu do you have any information about this brand? (any information, screenshot device or page)

Comment (liviuconcioiu) [reply Parser/Device/AbstractDeviceParser.php:762]: @sanchezzzhak I don't have any information about it.

**Comment by sanchezzzhak** [comment]
@liviuconcioiu pls resolve conflict

**Review by sanchezzzhak — APPROVED** [review]

**Pull request merged**